packages/api/server.js

```diff
@@ -49,8 +61,8 @@ app.use(
   })
 );
 
-app.get('/api/repos', async (req, res) => {
-  const repository = new Repository(null);
+app.get('/protected/api/repos', async (req, res) => {
```

Comment:
Things are far enough along at this point, that I think we can simplify things by only protecting a few specific routes, like the routes we're adding that support deleting repositories and tests.

We shouldn't need to adda new prefix like this, I think instead we could do something like this:

```
app.get('/api/repos', isLoggedIn.isLoggedIn, async (req, res) => {
  ....
})
```

Each route can have middleware optionally added as the next parameter, it then works like a chain where one need to finish before the next.